components/_patterns/01-atoms/02-text/text/02-blockquote.twig

```diff
@@ -14,7 +14,9 @@
 #}
 {% set blockquote_base_class = blockquote_base_class|default('blockquote') %}
 
-<blockquote{{ bem(blockquote_base_class, (modifiers), blockname) }}>
+<blockquote
+  {{ bem(blockquote_base_class, (blockquote_modifiers), blockquote_blockname) }}
+>
```

Comment:
This is fine, but is a case where I'd probably just keep it inline with `blockquote` since it's the only attribute we're defining